internal/flink/command_artifact_list.go

```diff
@@ -0,0 +1,55 @@
+package flink
+
+import (
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+	"github.com/confluentinc/cli/v3/pkg/output"
+)
+
+type customPluginOutList struct {
+	Id   string `human:"ID" serialized:"id"`
+	Name string `human:"Name" serialized:"name"`
+}
+
+func (c *command) newListCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "List Flink UDF artifacts.",
+		Args:  cobra.NoArgs,
+		RunE:  c.list,
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: "List Flink UDF artifacts in the org",
+				Code: "confluent connect artifact list",
+			},
+		),
+	}
+
+	pcmd.AddContextFlag(cmd, c.CLICommand)
+	pcmd.AddOutputFlag(cmd)
+
+	return cmd
+}
+
+func (c *command) list(cmd *cobra.Command, _ []string) error {
+	plugins, err := c.V2Client.ListCustomPlugins()
+	if err != nil {
+		return err
+	}
+
+	list := output.NewList(cmd)
+	for _, plugin := range plugins {
+		if strings.ToLower(*plugin.ConnectorType) != "flink-udf" {
```

Comment:
Shouldn't the backend return a consistent format? Why do we need to lower-case this?
```suggestion
		if strings.ToLower(plugin.GetConnectorType()) != "flink-udf" {
```